EIPS/eip-3473.md

```diff
@@ -450,18 +450,5 @@ There are no known security considerations for this EIP. More security considera
 
 ### License 
 
-###### apache 2.0
-
-    Copyright 2020 Sigmoid <info@SGM.finance>
-    
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-    
-    http://www.apache.org/licenses/LICENSE-2.0
-    
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
+###### CC0

```

Comment:
Please use the exact same text from the EIP template.

Reply:
hi please check the last version(e11261a), the license part is changed as it in the template

Reply:
suggestion applied